husky_description/urdf/dual_arm_husky.urdf.xacro

```diff
@@ -482,7 +482,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   </joint>
   <!-- Connect PTU to Husky bulkhead -->
   <joint name="husky_ptu_joint" type="fixed">
-    <origin rpy="0 0 0" xyz="0.365 0 0.221"/>
```

Comment:
@TheDash how did you originally come up with .221? We have just noticed the bracket that mounts the Bumblebee onto the PTU isn't incorporated in the model at all and hence the centre of rotation is off by ~3cm causing all images not to align :(